### ISSUE TITLE
Sequence file input validator does not accept gzipped files

### ISSUE THREAD

**Issue by sarahet** [seqan/seqan3#1591]
When defining an input file with a validator for sequence input files the following way:

```
parser.add_option(options.queryFile, 'q', "query", "Query sequences.",
seqan3::option_spec::REQUIRED,
seqan3::input_file_validator<seqan3::sequence_file_input<>>{});
```
it does not accept ```.gz``` files but only ```[embl,fasta,fa,fna,ffn,faa,frn,fastq,fq,genbank,gb,gbk,sam]```

Shouldn't this be possible as most sequence input files are actually compressed?

**Comment by rrahn** [comment]
Hey @sarahet thanks for the report.

This is currently indeed not supported and would be a feature. It does need some thoughts though to do it correctly. We will discuss this and see how we can put this into out road map.

**Comment by sarahet** [comment]
Great, thanks. A related issue would be that input validators generally do not have the possibility to allow for file endings hat include multiple dots, like ```.fastq.gz```. Could that be included as well or is that rather not planned? Related https://github.com/seqan/lambda/issues/161

**Comment by rrahn** [comment]
yes, I guess that is somewhat related. is the multiple dot version restricted to 2 dots assuming one valid extension and one compression extension? Or are more compex cases possible?

**Comment by sarahet** [comment]
So far I can only think of one extension and a compression extension, but of course there could be other use cases, that I currently don't have in mind ..

**Comment by smehringer** [comment]
Is it a use-case to compress you file multiple times, e.g. `fasta.gz.bz2`? (I don't even know if this makes sense, just wondering)

**Comment by rrahn** [comment]
I think `.fasta.tar.bz` etc. could be. Not that we would support that at the moment but maybe interesting for the future
